ddl/column.go

```diff
@@ -1359,8 +1359,9 @@ func (w *updateColumnWorker) BackfillDataInTxn(handleRange reorgBackfillTask) (t
 		taskCtx.nextKey = nextKey
 		taskCtx.done = taskDone
 
-		warningsMap := make(map[errors.ErrorID]*terror.Error, len(rowRecords))
-		warningsCountMap := make(map[errors.ErrorID]int64, len(rowRecords))
+		// Optimize for few warnings!
```

Comment:
I think it is better to avoid reserving too much memory that may not be needed.
On the other hand it will be a single allocation instead of multiple ones when resizing the maps.